jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/LocalRegistry.java

```diff
@@ -96,14 +88,11 @@ protected void before() throws IOException, InterruptedException {
               "-e",
               "REGISTRY_AUTH_HTPASSWD_REALM=Registry Realm",
               "-e",
-              "REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd",
-              "registry:2")
-          .run();
-    }
-
-    if (pullBusyBox) {
-      pullAndPushToLocal("busybox", "busybox");
+              "REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd"));
     }
+    dockerTokens.add("registry:2");
+    String[] tokenArray = new String[dockerTokens.size()];
+    new Command(dockerTokens.toArray(tokenArray)).run();
```

Comment:
I think we should just add a constructor that takes a List rather than doing this the round-about-array-way